website/docs/d/globalRole.html.markdown

```diff
@@ -27,6 +27,7 @@ data "rancher2_global_role_binding" "foo" {
 ## Attributes Reference
 
 * `id` - (Computed) The ID of the resource (string)
+* `group_principal_id` - (Computed) The group principal ID to assign global role binding. Rancher v2.4.0 or higher is required (string)
```

Comment:
Why can I use `user_id` but here we need `_principal_`

Reply:
This is how is defined at API, https://github.com/rancher/types/blob/master/client/management/v3/zz_generated_global_role_binding.go#L28

Reply:
Added examples on globlaRole resource doc.